bundles/pcmjava/tools.vitruv.applications.pcmjava.pojotransformations/src/tools/vitruv/applications/pcmjava/pojotransformations/java2pcm/Java2PcmClassifier.reactions

```diff
@@ -182,7 +182,7 @@ routine createOrFindSystem(java::Package javaPackage, String name) {
 	 match {
 		require absence of pcm::System corresponding to javaPackage
 		val foundSystem = retrieve optional pcm::System corresponding to SystemPackage.Literals.SYSTEM
-			with foundSystem.entityName.toFirstLower == javaPackage.name // PCM systems can be both upper and lower case
+			with foundSystem.entityName.toFirstLower == javaPackage.name.toFirstLower // PCM systems and Java packagescan be both upper and lower case
```

Comment:
typo in comment

Reply:
Thanks. Became obsolete due to changes w.r.t. your other comment.